docs/content/guides/rows/row-height.md

```diff
@@ -28,11 +28,14 @@ Configure row heights, using a number, an array or a function. Let your users ma
 
 ## Overview
 
-By default, the height of a row adjusts to the height of the content. The minimum height is `23px`. The row height can be passed as a `constant`, an `array`, or a `function`.
+By default, the height of a row adjusts to the height of the content. The content gets wrapped if it doesn't fit the cell's size. The default (and minimum) row height is 23 px (22 px + 1 px of the row's bottom border).
 
-The content inside a cell gets wrapped if it doesn't fit the cell's size.
+If your rows have different heights than the default 23 px (because you use multiline text, or [custom renderers](@/guides/cell-functions/cell-renderer.md#), or custom styles), do one of the following:
```

Comment:
Maybe we could add a final clarification:

```suggestion
If your rows have different heights than the default 23 px (because you use multiline text, or [custom renderers](@/guides/cell-functions/cell-renderer.md#), or custom styles), Handsontable might have a problem to lay out the grid correctly unless you do one of the following:
```

Reply:
@warpech – got it – I'd put it like this then: [c86cb5d](https://github.com/handsontable/handsontable/pull/10099/commits/c86cb5d5ba6b26d8b512b6f4c04878ddbbf55c77)